src/Sulu/Component/Content/Document/Behavior/LocalizedAuthorBehavior.php

```diff
@@ -16,6 +16,29 @@
  */
 interface LocalizedAuthorBehavior extends LocalizedBlameBehavior
 {
+    /**
+     * Returns lastModifiedEnabled.
+     *
+     * @return bool
+     */
+    public function getLastModifiedEnabled();
+
+    /**
+     * Returns lastModified-date.
+     *
+     * @return \DateTime|null
+     */
+    public function getLastModified();
+
+    /**
+     * Set lastModified-date.
+     *
+     * @param \DateTime|null $lastModified
+     *
+     * @return void
+     */
+    public function setLastModified($lastModified);
+
```

Comment:
This is a bc break, We need to create a new interface for this `LocalizedLastModifiedBehavior`.